src/services/usersService.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+let users = [];
+
+const getInitial = () => {
+  users = [];
+
+  return users;
+};
+
+const getAllUsers = () => users;
+
+const getUserById = (userId) => {
+  const foundUser = users.find(user => user.id === +userId);
+
+  return foundUser || null;
+};
+
+const createUser = (name) => {
+  const newUser = {
+    id: Math.max(...users.map(user => user.id), 0) + 1,
```

Comment:
Good stuff, but remember about DRY. This `Math.max(...data.map(...` logic is the same for both users and expenses. Move it to function and place it in separate `utils` folder.